packages/clarity-js/src/data/encode.ts

```diff
@@ -112,6 +112,8 @@ export default function(event: Event): void {
                 tokens.push(e);
                 tokens.push(extract.data[e]);
             }
+            tokens.push('u')
```

Comment:
We shouldn't hardcode this and include it as part of the interface: [ExtractData](https://github.com/microsoft/clarity/blob/master/packages/clarity-js/types/data.d.ts#L395)